rqt_py_common/src/rqt_py_common/layout_util.py

```diff
@@ -39,7 +39,7 @@
 
 
 class LayoutUtil(object):
-    _logger = logging.get_logger("LayoutUtil")
```

Comment:
I mentioned in the other ticket that this was just one case. The other locations calling the same function still use double quotes.